src/excmds.ts

```diff
@@ -177,10 +184,12 @@ export async function getinput() {
  */
 //#background
 export async function editor() {
-    let url = new URL((await activeTab()).url)
+    let tab = await activeTab()
+    let selector = await Messaging.messageTab(tab.id, "excmd_content", "getInputSelector", [])
+    let url = new URL(tab.url)
     if (!await Native.nativegate()) return
     const file = (await Native.temp(await getinput(), url.hostname)).content
-    fillinput((await Native.editor(file)).content)
+    Messaging.messageTab(tab.id, "excmd_content", "fillinput", [selector, (await Native.editor(file)).content])
```

Comment:
Why the roundabout fashion? Does selector get mangled?

Reply:
I'm not sure I understand your question. If you're asking why I'm using `Messaging.messageTab` rather than just calling `fillinput()`, it's because `fillinput()` will be called in whatever tab is currently active, which might not be the tab the editor was spawned from.

Reply:
Ah, good thinking : )